src/runtime/entitlements-manager.js

```diff
@@ -199,19 +202,24 @@ export class EntitlementsManager {
       .eventManager()
       .logSwgEvent(AnalyticsEvent.EVENT_UNLOCKED_BY_METER, false);
 
+    const token = parseQueryString(this.win_.location.search)['gaa_n'];
```

Comment:
I noticed this is repeated on line 255. Would it make sense to make this a private method, like `getToken_()`?

Reply:
Added a getGaaToken_ function.